src/io/matpower.jl

```diff
@@ -73,7 +73,7 @@ end
 
 
 function parse_matpower(file_string)
-    data_string = readall(open(file_string))
+    @compat data_string = readstring(open(file_string))
```

Comment:
Just curious: I used a force push to get rid of those changes, so now when I click "View changes" above your comment, the previous commit is gone. Is there a better way to modify a commit in an PR? I hesitate to add another commit just to undo a bit of the last one...